src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int MAX_VALUE = 102;
```

Comment:
For now, maximum possible random number is 101, but the task says it should be 100. How do you think we can fix that?